easybuild/easyconfigs/o/Octave/Octave-4.0.0-foss-2016a.eb

```diff
@@ -0,0 +1,58 @@
+easyblock = 'ConfigureMake'
+
+name = 'Octave'
+version = '4.0.0'
+
+homepage = 'http://www.gnu.org/software/octave/'
+description = """GNU Octave is a high-level interpreted language, primarily intended for numerical computations."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [
+    ('Bison', '3.0.4'),
+    ('flex', '2.6.0'),
+    ('Autotools', '20150215'),
+    ('libtool', '2.4.6'),
+    ('gperf', '3.0.4'),
+]
+
+dependencies = [
+    ('libX11', '1.6.3'),
+    ('libXext', '1.3.3'),
+    ('libXft', '2.3.2'),
+    ('PCRE', '8.38'),
+    ('ncurses', '6.0'),
+    ('libreadline', '6.3'),
+    ('arpack-ng', '3.3.0'),
+    ('cURL', '7.47.0'),
+    ('FLTK', '1.3.3'),
+    ('fontconfig', '2.11.94'),
+    ('freetype', '2.6.2'),
+    ('GLPK', '4.58'),
+    ('GL2PS', '1.3.9'),
+    ('gnuplot', '5.0.3'),
+    ('Java', '1.8.0_72', '', True),
+    ('zlib', '1.2.8'),
+    ('Mesa', '11.1.2'),
+    ('libGLU', '9.0.0'),
+    ('Qhull', '2015.2'),
+    ('Qt', '4.8.7'),
+    ('HDF5', '1.8.16', '-serial'),
+    ('qrupdate', '1.1.2'),
+    ('SuiteSparse', '4.5.1', '-METIS-5.1.0'),
+    ('GraphicsMagick', '1.3.23'),
+]
+
+configopts = 'MOC=$EBROOTQT/bin/moc --with-blas="$LIBBLAS" --with-lapack="$LIBLAPACK" '
```

Comment:
@boegel from  https://www.gnu.org/prep/standards/html_node/Configuration.html
Specifying variables as arguments to configure, like this:
```
./configure CC=gcc
```
is preferable to setting them in environment variables:
```
CC=gcc ./configure
```
as it helps to recreate the same configuration later with config.status. However, both methods should be supported.